TESTS/netsocket/tls/tlssocket_handshake_invalid.cpp

```diff
@@ -32,7 +32,7 @@ void TLSSOCKET_HANDSHAKE_INVALID()
     TEST_ASSERT_EQUAL(NSAPI_ERROR_OK, sock.open(NetworkInterface::get_default_instance()));
     TEST_ASSERT_EQUAL(NSAPI_ERROR_OK, sock.set_root_ca_cert(tls_global::cert));
     TEST_ASSERT_EQUAL(NSAPI_ERROR_NO_CONNECTION,
-                      sock.connect("google.com", MBED_CONF_APP_ECHO_SERVER_DISCARD_PORT_TLS));
+                      sock.connect("os.mbed.com", MBED_CONF_APP_ECHO_SERVER_DISCARD_PORT_TLS));
```

Comment:
@michalpasztamobica. @SeppoTakalo  there was a reason to use google.com?

Reply:
This might have something to do with IPv6 address resolution, see the comment https://github.com/ARMmbed/mbed-os/pull/9387#issuecomment-458485510

Reply:
Yes, os.mbed.com does not accept IPv6 traffic which means it will fail with a different error than we expect. Google.com accepts Ipv6 traffic but will fail due to invalid certificate (which is what we want to test).